encord/objects/ontology_labels_impl.py

```diff
@@ -2582,24 +2571,27 @@ def _create_new_classification_instance(
 
         return None
 
-    # This is only to be used by non-frame modalities (e.g. Audio)
-    def _create_new_classification_instance_with_ranges(
+    def _create_new_classification_instance_from_answer(
         self, classification_answer: ClassificationAnswer
-    ) -> ClassificationInstance:
+    ) -> ClassificationInstance | None:
+        """Create a ClassificationInstance from a classification answer if sufficient metadata is present"""
+        if not _is_containing_metadata(classification_answer):
+            return None
+
         feature_hash = classification_answer["featureHash"]
         classification_hash = classification_answer["classificationHash"]
 
         label_class = self._ontology.structure.get_child_by_hash(feature_hash, type_=Classification)
 
         range_view = ClassificationInstance.FrameData.from_dict(classification_answer)
+        is_data_type_range_only = not is_geometric(self.data_type)
 
         classification_instance = ClassificationInstance(
-            label_class, classification_hash=classification_hash, range_only=True
```

Comment:
This was the bug. 'modern' classifications have metadata at the classification instance level, which would then fall into this branch, which was previously reserved for range-only classifications